app/services/authcore/authcore.ts

```diff
@@ -40,6 +41,16 @@ function parseAuthCoreUser({
   }
 }
 
+function findBestAvailableLanguage() {
+  const locale = i18n.currentLocale()
```

Comment:
what is diff between `.locale` and `currentLocale()` ?
former was used here https://github.com/likecoin/likecoin-app/blob/c88500e56797abc1ecc82d86eb774cf07c63b9e0/app/screens/sign-in-screen/sign-in-screen.tsx#L192

Reply:
`currentLocale()` will fallback to `defaultLocale` if `locale` is undefined
I don't think it has much difference here, I shall change it to preserve the consistency